setup.py

```diff
@@ -18,7 +18,7 @@
         'pytest==6.0.1',
         'pylint==2.5.3',
         'requests-oauthlib==1.3.0',
-        'wikia-common-kibana==2.2.6',
+        'wikia-common-kibana==2.2.7',
```

Comment:
@mszabo-wikia, you can use https://github.com/macbre/elasticsearch-query/releases/tag/v2.4.0 which I forked some time ago :slightly_smiling_face: